examples/examples/futures-spawn.rs

```diff
@@ -1,41 +0,0 @@
-#![deny(rust_2018_idioms)]
```

Comment:
This file can be deleted because we've already re-implemented it in `nightly-examples`.